src/git/Repository.ts

```diff
@@ -167,12 +167,21 @@ export class Repository {
         });
     }
 
-    public static getLatestTagOfPattern(repoName: string, repoUrl: string, tagPattern: string): Promise<string> {
+    public static getLatestTagOfPattern(repoName: string, repoUrl: string, tagPattern: string, rootDir: string): Promise<string> {
         return new Promise<string>((resolve, reject) => {
             Global.isVerbose() && console.log(`[${repoName}]: Getting the last tag with pattern ${tagPattern}:\n`);
-            simpleGit().listRemote(['--tags', '--refs', '--sort=version:refname', repoUrl, tagPattern], (err, result: string) => {
+            let useRepoUrl = repoUrl;
+            if (repoUrl.startsWith('git@')) {
+                const localOriginUrl = this.getLocalOriginUrl(repoName, rootDir);
+                if (localOriginUrl.startsWith('https:')) {
```

Comment:
Hm `localOriginUrl` is a `Promise`. I guess  a `Promise` don't have a `startsWith` method.